package.json

```diff
@@ -16,11 +16,13 @@
     "notification"
   ],
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "files": [
     "lib"
   ],
   "scripts": {
-    "build": "tsc",
+    "flowgen": "flowgen lib/index.d.ts -o lib/index.js.flow && flowgen lib/src/toast-banner-component.d.ts -o lib/src/toast-banner-component.js.flow && flowgen lib/src/toast-banner-container.d.ts -o lib/src/toast-banner-container.js.flow",
```

Comment:
Could you check if there is a way to do it on all files, not manually? 
Considering this module is very beta and will have more files soon.

Reply:
There isn't a possible solution in `flowgen`. Therefore I've made the script `flow.sh` :)

Reply:
Awesome! Thanks 🙌